### ISSUE TITLE
code changes on LTA task

### ISSUE THREAD

**Issue by CeeZh** [EGO4D/forecasting#27]
Hi! 

I see a significant performance increase from Host_5075_Team (Slowfast-trf v1) to Host_5075_Team (Slowfast-trf v2) on LTA challenge leaderboard. Could you provide the code or hparams changes? Thanks!

**Comment by Tushar-N** [comment]
No code changes here! The model is re-trained on the expanded v2 dataset (test set is the same). See relevant announcement [here](https://discuss.ego4d-data.org/t/ego4d-v2-0-release-updates/197).

**Comment by CeeZh** [comment]
Thanks for your answer!

Are the provided LTA checkpoints pretrained on the v1 dataset or v2 dataset?
